webpack.config.js

```diff
@@ -24,10 +24,6 @@ const commonConfig = {
                     ]
                 },
                 test: /\.js$/
-            },
-            {
-                use: 'node-loader',
```

Comment:
Since we are using `jitsi-meet-electron-utils` package as externals. We no longer need `node-loader`.